CalcualtorLogic.js

```diff
@@ -114,6 +114,43 @@ class Calculator {
         div4.appendChild(periodButton);
         div4.appendChild(negativeValueButton);
         wrapperDiv.appendChild(div4);
+        const div5 = document.createElement("div");
+        const multiplyingButton = document.createElement("button");
+        multiplyingButton.dataset.number = "*";
```

Comment:
this need to be `multiplyingButton.dataset.operator` 
ditto for the rest of the operator buttons